CpEpEngine.cpp

```diff
@@ -2577,4 +2578,16 @@ STDMETHODIMP CpEpEngine::SignatureVerifies(BSTR text, BSTR signature, VARIANT_BO
     }
     
     return status;
-}
\ No newline at end of file
+}
+
+STDMETHODIMP CpEpEngine::ManagePassphrase(LPSAFEARRAY accounts_with_old_passphrases, BSTR new_passphrase, LPSAFEARRAY* error_accounts) {
+    return PEP_ILLEGAL_VALUE;
+}
+
+STDMETHODIMP CpEpEngine::HasPassphrase(BSTR account, VARIANT_BOOL* result) {
+    return PEP_ILLEGAL_VALUE;
+}
+
+STDMETHODIMP CpEpEngine::UnlockKeysWithPassphrase(LPSAFEARRAY accounts, LPSAFEARRAY *error_accounts) {
+    return PEP_ILLEGAL_VALUE;
+}
```

Comment:
@planckdz I am wondering if it makes sense to include code that returns different values based on different hard-coded test users, so the UI can prepare for the different use cases right from the beginning.

Reply:
@planckdz i agree with @ip-plank-security , it would be helpful to have some hard-coded accounts which return different values.

Reply:
Those two I can manage without the sequoia backend. At the core level, HasPassphrase works, and it's not unlikely that UnlockKeysWithPassphrase will be done by tomorrow. Then the matching wrapper logic and they are live, provided to switch to the feature branches.

If you want hard-coded values, please provide them and I will put them here. Some gherkin tables?

HasPassphrase: | account | result |
UnlockKeysWithPassphrase: | account | is_error |

Reply:
then let's be reasonable - if you progress quite well, let's put energy into actual value. we then start to build UI with the methods that will fully work